packages/core/src/test/credentials/utils.test.ts

```diff
@@ -11,23 +11,23 @@ import globals from '../../shared/extensionGlobals'
 
 describe('ExtensionUse.isFirstUse()', function () {
     let instance: ExtensionUse
+    const notHasExistingConnections = () => false
```

Comment:
should this be `hasNoExistingConnections` ?

Reply:
Makes sense, I'll follow up with the name change after. Don't want to have to run CI again unless theres something critical.